cloud/pkg/cloudhub/servers/udsserver/server.go

```diff
@@ -0,0 +1,75 @@
+package udsserver
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"k8s.io/klog"
+
+	ctx "github.com/kubeedge/beehive/pkg/core/context"
+	"github.com/kubeedge/beehive/pkg/core/model"
+	hubmodel "github.com/kubeedge/kubeedge/cloud/pkg/cloudhub/common/model"
+	"github.com/kubeedge/kubeedge/cloud/pkg/cloudhub/common/util"
+	"github.com/kubeedge/kubeedge/common/constants"
+)
+
+// StartServer serves
+func StartServer(config *util.Config, c *ctx.Context) {
```

Comment:
please add the return type;
you have multiple return clauses with an value and didn't set a return type